cpp/src/io/orc/dict_enc.cu

```diff
@@ -67,7 +67,7 @@ void rowgroup_char_counts(device_2dspan<size_type> counts,
   CUDF_CUDA_TRY(
     cudaOccupancyMaxPotentialBlockSize(&min_grid_size, &block_size, rowgroup_char_counts_kernel));
   auto const grid_size =
-    dim3(cudf::util::div_rounding_up_unsafe<unsigned int>(num_rowgroups, block_size),
+    dim3(cudf::util::div_rounding_up_safe<unsigned int>(num_rowgroups, block_size),
```

Comment:
Why changing from `unsafe` to `safe`? A bug?

Reply:
I changed this during a debugging. I left it since it's safe. This may be reverted.